docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.3'
+
+services:
+
+  backend:
+    build: ./backend
+    expose:
+      - 5000
+    environment:
+      PORT: 5000
+      DATABASE_HOST: postgres-dev-db
+    ports:
+      - 5000:5000
+    volumes:
+      - ./backend/src:/backend/src
+    command: npm run start:dev
+    depends_on:
+      - postgres-dev-db
+
+  front-end:
+    build: ./front-end
+    expose:
+      - 3000
+    ports:
+      - 3000:3000
+    volumes:
+      - ./front-end/src:/front-end/src
+      - ./front-end/public:/front-end/public
+    links:
+      - backend
+    command: npm run start
```

Comment:
Is this needed? Doesn't the Dockerfile have CMD with npm start?

Reply:
yes thank you . it basically does the same as the Dockerfile. I have removed it.